### DISCUSSION TITLE
Ajouter plusieurs code GA4

### DISCUSSION THREAD

**Discussion by Steuphh** [AmauriC/tarteaucitron.js#1255] [Q&A]
Bonjour et merci pour ce sympathique script compatible avec le consent mode v2.

Je me pose la question car je n'ai pas trouvé si il était possible d'ajouter deux code GA-XXXXXX pour le même site?
Mes différents essais ont malheureusement été infructueux.

Merci par avance

Stéphane

**Comment by AmauriC** [comment]
Bonjour Stéphane : )

Il y a le service "multiple gtag" pour ça:

```
<script type="text/javascript">
tarteaucitron.user.multiplegtagUa = ['UA-XXXXXXXX-X', 'UA-XXXXXXXX-X', 'UA-XXXXXXXX-X'];
(tarteaucitron.job = tarteaucitron.job || []).push('multiplegtag');
</script>
```

**Comment by Steuphh** [comment]
Ha merci beaucoup.
Comme un boulet, je croyais que c'était que pour les anciens Analytics en UA...

**Reply by AmauriC** [reply]
Je comprend tout à fait la confusion, je note de le renommer en GA4 lui aussi 👍
